### PR TITLE
Transpile ES6->ES5

### PR DESCRIPTION
**Pull request by dternyak**

### CODE REVIEW

File: webpack.cordova.config.js [hunk 1]
```diff
@@ -70,7 +70,9 @@ module.exports =
 				loader: 'babel-loader',
 				options: {
 					cacheDirectory: false,
+                    presets: ["es2015"],
```
Comment (dternyak) [inline webpack.cordova.config.js]: weird in the diff but normal if you actually open it. not sure why.

**Comment by paulshapiro** [comment]
Hey dude,

Thanks for submitting this :) Just one thing. Could we remove the slightly scary merge commit? :) Might require starting from scratch, but that looks like it wouldn't be a huge deal.

Let me know

File: webpack.cordova.config.js (hunk 1 above)
Comment (paulshapiro) [inline webpack.cordova.config.js]: spaces vs tabs

Comment (paulshapiro) [inline webpack.cordova.config.js]: went with tabs instead of spaces cause it's a js (vs json) file

**Comment by dternyak** [comment]
Closing in favor of https://github.com/mymonero/mymonero-app-js/pull/105

**Pull request closed without merging**